### ISSUE TITLE
bug: unit test fails with RLN dependency error on apple machine

### ISSUE THREAD

**Issue by chaitanyaprem** [waku-org/go-waku#659]
**Describe the bug**

Unit tests failing wrt RLN dependency error. Looks like dependent repo is not updated.
@richard-ramos , I remember we saw a similar issue happen few weeks ago as well. Can we somehow automate/link this so that we don't see this happening frequently? I am sure it is due to manual steps that we are facing this issue.

**To Reproduce**
Run `make test` from master branch

**Expected behavior**
No test failures should occur on master

**Screenshots**

```
# github.com/waku-org/go-zerokit-rln-apple/rln
../../go/pkg/mod/github.com/waku-org/go-zerokit-rln-apple@v0.0.0-20230807124901-05ef8aca570d/rln/wrapper.go:50:89: not enough arguments in call to (_Cfunc_new_with_params)
	have (_Ctype_ulong, *_Ctype_struct_Buffer, *_Ctype_struct_Buffer, *_Ctype_struct_Buffer, **_Ctype_struct_RLN)
	want (_Ctype_ulong, *_Ctype_struct_Buffer, *_Ctype_struct_Buffer, *_Ctype_struct_Buffer, *_Ctype_struct_Buffer, **_Ctype_struct_RLN)
```

### go-waku version/commit hash
Verified at commit : 8a9c4d68e3551587209b9fbb5386b5fb3d34cc5a

Note: Looks like this is happening on Mac M1 as this issue is not noticed in CI.

**Comment by chaitanyaprem** [comment]
Upon little debugging, i have noticed that https://github.com/waku-org/go-zerokit-rln-apple repo was not updated with changes to API code which is causing this error. 
It should have been done as part of #619.

**Comment by chaitanyaprem** [comment]
Fixed by #662
